crawler/crawler.py

```diff
@@ -338,6 +361,10 @@ def start_autonomous_crawler(num_processes, logfile):
                             pid)
                     del jobs[index]
             time.sleep(0.1)
+	
+        logger.info('terminating docker-monitor process.(pid=%s)'%(eventProc.pid))
+        eventProc.terminate()
```

Comment:
Trying to make sure that there won't be eventProc's leaked under any possible case. For exmaple, if any worker throws some unhandled exception and die, this will always be called right? this `terminate()`?